src/replit_river/client.py

```diff
@@ -46,6 +50,15 @@ def translate_unknown_value(
         return RiverUnknownValue(tag="RiverUnknownValue", value=value)
 
 
+def translate_unknown_error(
+    value: Any, handler: Callable[[Any], Any], info: ValidationInfo
+) -> Any | RiverUnknownError:
+    try:
+        return handler(value)
+    except Exception:
+        return RiverUnknownError(code=ERROR_CODE_UNKNOWN, message="Unknown error")
```

Comment:
We have some guarantees about error values

```suggestion
    value: Any, handler: Callable[[Any], Any], info: ValidationInfo
) -> Any | RiverUnknownError:
    try:
        return handler(value)
    except Exception:
        if isinstance(value, dict) and "code" in value and "message" in value:
            return RiverUnknownError(
                code=value["code"],
                message=value["message"],
            )
        else:
            return RiverUnknownError(code=ERROR_CODE_UNKNOWN, message="Unknown error")
```